notes/Module-2/2.4.6-OpenID_Connect.md

```diff
@@ -0,0 +1,18 @@
+# Coach Notes
+
+## Coach comments
+Please add comments below which will help future coaches delivering this topic or suggestions for improvement of the content.
+
+|**Coach name**|**Delivery date**|**Comments**|
+|--------------|-----------------|------------|
+|Mandy Warren|19/05/2020|Learning objectives not correct?|
+|Mandy Warren|19/05/2020|Just want to confirm - is this a separate web app being spun up on a different port? Am assuming it is and then it will make calls to the Banking app APIs running on a different server? We went down the route of creating an SPA instead but I think Regular Web App probably easier unless you familiar with React (or we given them option for either). |
+|Mandy Warren|19/05/2020|Text after "OpenID connect" a bit confusing and does "Summary" seems to refer to the assignment as it mentions banking?|
+|Mandy Warren|19/05/2020|There are 2 assignments here - better to have the Banking one as the end of unit assessment (instead of a quiz) so we have a way to assess their knowledge. I gave this as an individual assessment as a) the group were all working in different languages and b) they wanted to work at their own pace and c) I needed to have evidence for how individuals were progressing. Also, although the old notes mention to NEVER commit passwords to version control, the new notes don't. It worries me having gmail account details in the code/in GitHub/on a Cloud server which gets forgotten about hence I did not get my cohort to do the emailing. I also asked them to produce a sequence diagram (including the User, the Banking App, Auth0 and the BankingAPI) which illustrated the process of logging in and sending a token to access the Banking APIs. This really helped their understanding and we could validate the design before they started coding.|
```

Comment:
You can make these individual assignment.

Reply:
done